pxr/imaging/hdMtlx/hdMtlx.cpp

```diff
@@ -562,6 +562,12 @@ HdMtlxCreateMtlxDocumentFromHdMaterialNetworkInterface(
 
     // Initialize a MaterialX Document
     mx::DocumentPtr mxDoc = mx::createDocument();
+
+    // Need to figure out how to read the actual value from the netInterface
+    std::string materialXVersionString = "1.37";
```

Comment:
@ld-kerley , should this default value be 1.38? Or maybe it's just a temporary/filler value that doesn't matter?

Reply:
Yes - this is place holder - I'm hoping that someone with a bit more intimate hydra understanding can guide me as to the right way to get the real MaterialX version string from USD here.